src/components/JoinPage/JoinContent.js

```diff
@@ -0,0 +1,139 @@
+import React from "react"
+import styled from "styled-components"
+
+import { lightenDarkenColor, SB_NAVY } from "@colors"
+import SquareLogo from "components/squareLogo"
+import { capitalize } from "utils/string"
+
+const Wrapper = styled.div`
+  border-left: 6px solid ${props => props.color};
+  transition: border-left 0.3s;
+  padding-left: 1em;
+  margin: 2em 0;
+  @media (min-width: 1000px) {
+    padding-left: 1.5em;
+    margin: 4em 8em;
+  }
+`
+
+const ContentHeaderWrapper = styled.h3`
+  display: flex;
+  justify-content: space-between;
+  font-size: 1.8em;
+  margin-bottom: 0;
+`
+
+const HeaderText = styled.div`
+  font-weight: 500;
+`
+
+const ContentBody = styled.div`
+  line-height: 1.3em;
+  font-size: 1.2em;
+  @media (min-width: 600px) {
+    font-size: 1.3em;
+  }
+`
+
+const QualitiesHeader = styled.h2`
+  letter-spacing: 0.15em;
+  text-transform: uppercase;
+  text-align: center;
+  font-weight: 500;
+  font-size: 1.8em;
+  max-width: 500px;
+  margin: auto;
+
+  @media (min-width: 600px) {
+    font-size: 2em;
+  }
+`
+
+const QualitiesWrapper = styled.ul`
+  margin: 1em auto 2em;
+  padding: 0;
+  max-width: 450px;
+  font-size: 1.2em;
+  line-height: 1.5em;
+
+  @media (min-width: 600px) {
+    font-size: 1.3em;
+    padding: 0 40px;
+  }
+`
+
+const Quality = styled.li`
```

Comment:
When the quality overflows to two lines, it looks kind of awkward? I think maybe the second line needs to be indented?

Reply:
I actually consciously left this here, styling lists is sorta tough and I think I need to do some weird overflow stuff to make it look nice so I just made a good vs. perfect decision. If it looks really awkward I can fix it though

Reply:
Yeah I think it's good enough to publish

Reply:
can it just be a little wider? The devops page is weird, with every sentence having 1 word overflowing

Reply:
<img width="560" alt="Screen Shot 2019-05-19 at 6 25 32 PM" src="https://user-images.githubusercontent.com/13836027/57989052-8f894600-7a63-11e9-9fb4-625a352b3d7f.png">

It being wider makes the short sentences in the developer page look off-center.

Reply:
I'd rather fluff the sentences a little